docs/reference/setup/install/docker.asciidoc

```diff
@@ -82,7 +82,7 @@ output to the terminal, plus an enrollment token for enrolling {kib}.
 ifeval::["{release-state}"!="unreleased"]
 [source,sh,subs="attributes"]
 ----
-docker run --name es-node01 -p 9200:9200 -p 9300:9300 -it {docker-image}
+docker run --name es-01 --net elastic -p 9200:9200 -p 9300:9300 -it {docker-image}
```

Comment:
AFAIK names are `es01`, etc — rather than `es-01`. Does it sound good to you?

Reply:
Yes! Thanks for pointing out that discrepancy. I just pushed a change to remove the hyphen in 701798e4c2965b0462012f525979c91468c0e203